File: ts/iq/urn-xmpp-whatsapp-dirty~set~clean.ts
/** <iq type="set" xmlns="urn:xmpp:whatsapp:dirty">. Replies with one of: Success, Error. */
export function buildDirtyBitsClean(): BinaryNode {
  return { tag: 'iq', attrs: { to: 's.whatsapp.net', xmlns: 'urn:xmpp:whatsapp:dirty', type: 'set' }, content: [
      { tag: 'clean', attrs: {  } },
    ] }
}

File: rust/iq/urn-xmpp-whatsapp-dirty~set~clean.rs
/// <iq type="set" xmlns="urn:xmpp:whatsapp:dirty">. Replies with one of: Success, Error.
pub fn build_dirty_bits_clean() -> Node {
    NodeBuilder::new("iq")
        .attr("to", "s.whatsapp.net")
        .attr("xmlns", "urn:xmpp:whatsapp:dirty")
        .attr("type", "set")
        .child(NodeBuilder::new("clean").build())
        .build()
}